docs/articles/mirroring.md

```diff
@@ -0,0 +1,432 @@
+<li><p><code>true</code>: Remove the source prefix.</p>
+<div class="note">
+<p><strong>Note:</strong> If the source prefix was specified with meta-characters (such as <strong>**</strong>), only the prefix segments that precede the meta-characters are removed. Any remaining path segments are appended to the destination path.</p>
+</div></li>
+</ul></td>
+</tr>
+</tbody>
+</table>
+
+## Example: Multiple repositories with polled mirroring
+
+The following is an example of sync configuration for mirroring multiple repositories with polled mirroring.
+
+```
+"sync": {
+  "enable": true,
+  "credentialsFile": "./examples/sync-auth-filepath.json",
+  "registries": [
+    {
+      "urls": ["https://registry1:5000"],
+      "onDemand": false,
+      "pollInterval": "6h",
+      "tlsVerify": true,
+      "certDir": "/home/user/certs",
+      "maxRetries": 3,
+      "retryDelay": "5m",
+      "onlySigned": true,
+      "content": [
+        {
+          "prefix": "/repo1/repo",
+          "tags": {
+            "regex": "4.*",
+            "semver": true
+          }
+        },
+        {
+          "prefix": "/repo2/repo",
+          "destination": "/repo2",
+          "stripPrefix": true
+        },
+        {
+          "prefix": "/repo3/repo"
+        }
+      ]
+    }
+  }
+```
+
+The configuration in this example will result in the following behavior:
+
+- Only signed images (notation and cosign) are synchronized.  
+- The sync communication is secured using certificates in `certDir`.
+- This registry synchronizes with upstream registry every 6 hours. 
+- On-demand mirroring is disabled.
+- Based on the content filtering options, this registry synchronizes these images:
+    - From /repo1/repo, images with tags that begin with "4." and are semver compliant. <br/>Files are stored locally in /repo1/repo on localhost.
+    - From /repo2/repo, images with all tags. <br/>Because `stripPrefix` is enabled, files are stored locally in /repo2. For example, docker://upstream/repo2/repo:v1 is stored as docker://local/repo2:v1.
+    - From /repo3/repo, images with all tags. <br/>Files are stored locally in /repo3/repo.
+
+## Example: Multiple registries with on-demand mirroring
+
+The following is an example of sync configuration for mirroring multiple registries with on-demand mirroring.
+
+```
+{
+  "distSpecVersion": "1.0.1",
+  "storage": {
+    "rootDirectory": "/tmp/zot",
+    "gc": true
+  },
+  "http": {
+    "address": "0.0.0.0",
+    "port": "8080"
+  },
+  "log": {
+    "level": "debug"
+  },
+  "extensions": {
+    "sync": {
+      "enable": true,
+      "registries": [
+        {
+          "urls": ["https://k8s.gcr.io"],
+          "content": [
+            {
+              "prefix": "**", 
+              "destination": "/k8s-images"
+            }
+          ],
+          "onDemand": true,
+          "tlsVerify": true
+        },
+        {
+          "urls": ["https://docker.io/library"],
```

Comment:
A small example calling out docker.io sync explicitly pls.
^ @peusebiu

Reply:
I did that below, right?
```skopeo copy --src-tls-verify=false docker://localhost:8080/docker-images/alpine <dest>```

Reply:
I can make it more explicit in that example like this:
"...causes zot to synchronize the content with the docker.io registry:"

And the next part as:
"...causes zot to synchronize the content with the gcr.io registry:"